### DISCUSSION TITLE
Categories / Tags

### DISCUSSION THREAD

**Discussion by Engholm** [akeeba/panopticon#333] [Ideas]
Hi,

I'm already pretty happy with this early stage of Panopticon and grateful, that this kind of selfhosted tool has been released. Good timing :)

An **idea / feature request**. Maybe it's already on the roadmap and I've overseen it:
It would be great to haven an option to set **Tags and/or Categories** for sites. At the moment I'm still using an external service to have an eye on around 200 Joomla Sites. That service offers an option to group sites with tags.

Best regards and thanks for this great tool!
Björn

**Comment by nikosdion** [comment]
There are groups, and each site can belong to one or more groups.

Currently, groups are used for access control.

If I understand you correctly, you want groups to be used to filter displayed sites?

**Comment by Engholm** [comment]
> If I understand you correctly, you want groups to be used to filter displayed sites?

Exactly. I haven't mention that filter option, but that's exactly what I've meant: Group sites with tags (or categories) + filter option for tags (or categories).

**Reply by nikosdion** [reply]
That's a good feature idea, and now it's implemented for the upcoming version 1.0.5 :)

**Comment by pAnd0rASBG** [comment]
Are Tags still coming? :)

**Reply by nikosdion** [reply]
As I had explained last year, we have Groups instead of tags. You can filter sites by group.
